### PR TITLE
Don't log null byte in replayserver.py

### LINKED ISSUE #648: Strange symbol in forever.log
When trying to copy paste the whole log I've noticed  that just part of it got pasted even thought I hit Ctrl A to select all. So when I went to the last line I found this there:
https://puu.sh/uuXaw/9206e69be6.png

Not sure how severe that is if all. The replay was a FAF Beta mod where I did some testing for Icecreamer.
When searching replay vault it is there but missing end time
https://puu.sh/uuXgk/d26a4f9c1b.png

### PR DESCRIPTION
**Pull request by Wesmania**
Fixes #648.

Signed-off-by: Igor Kotrasinski <ikotrasinsk@gmail.com>

- [x] PR branch should be named `issuenum`-`fix`/`feature`/`cleanup`-`description`
- [x] Code is split into logical commits
- [ ] Code has tests
- [x] Final commit includes "Fixes #issue" in commit message

When all builds pass and a maintainer is happy with your PR, the "ready" label will be applied. Please complete these tasks then:

- [ ] Rebase onto develop
- [ ] Add changelog entry
- [ ] Remove this entire template section

### CODE REVIEW

File: src/fa/replayserver.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def readDatas(self):
             #This prefix means "P"osting replay in the livereplay protocol of FA, this needs to be stripped from the local file            
             if data.startsWith("P/"):
                 rest = data.indexOf("\x00") + 1
-                self.__logger.info("Stripping prefix '" + str(data.left(rest)) + "' from replay.")
+                self.__logger.info("Stripping prefix '" + str(data.left(rest) - 1) + "' from replay.")
```
Comment (duk3luk3) [inline src/fa/replayserver.py]: I think you mean `data.left(rest - 1)`

Comment (duk3luk3) [reply src/fa/replayserver.py]: Also you could just change this all into

```python
prefix, tail = data.split("\x00", 1)
self.__logger.info("Stripping prefix '" + str(prefix) + "' from replay.")
self.replayData.append(tail)
```

Comment (Wesmania) [reply src/fa/replayserver.py]: We can't, unfortunately - it's a QByteArray, not a string.

Comment (duk3luk3) [reply src/fa/replayserver.py]: `prefix, tail = str(data).split("\x00", 1)` then?

Comment (Wesmania) [reply src/fa/replayserver.py]: It's later appended to another QByteArray - I don't know if we want to convert it to Python string and back just for some nice syntax.

Comment (Wesmania) [reply src/fa/replayserver.py]: Arguably it only happens once per replay, but that conversion still feels less tidy to me than just working with what QByteArray has.

**Pull request merged**